src/pages/components/tag/style.mdx

```diff
@@ -73,10 +89,10 @@ amount of content. All four corners of a tag are rounded with a 24px radius.
 
 ## Sizes
 
-| Size        | Height px / rem |
-| ----------- | --------------- |
-| Small (sm)  | 16 / 1          |
-| Medium (md) | 24 / 1.5        |
+| Size  | Height px / rem |
+| ----- | --------------- |
+| Small | 18px / 1.125    |
+| Big   | 24 / 1.5        |
```

Comment:
Can we keep it as Small & Medium? It makes sense to have it match our prop sizes (xs, md, lg, xl) for less confusion & consistency.

Reply:
We talked about this a while back and things that only have two sizes we wanted to use small and big. We do this in toggle as well. 

We decided on new terms and against the size props of small and medium because the value also don't match. The size for 24px things is actually `xs`. Small = 32px and medium = 40px.

Reply:
Yes but in the new sizing modal values correspond to a size
- xs = 24px
- sm = 32px
- md = 40px
- ect. 

So the name of the tag would be `xxs` (which doesn't exist) and `xs` to match that model which didn't make sense to the designers.

Reply:
Yes, I understand. I mean, match the specific component API. In the tag component the size props we offer are `sm` and `md` (md being the default).

Reply:
Oh is it actually defined at me now? I thought it was just 'tag' and 'small tag'. I didn't realize we've renamed the default to be medium

Reply:
Yes, we have the defined size props for tag: https://github.com/carbon-design-system/carbon/blob/6a345aabaecef2caaff74d3ddb2e9a503537896b/packages/react/src/components/Tag/Tag.js#L148-L152

I think you're thinking of toggle. We had a `ToggleSmall` that has been deprecated and the `Toggle` component now supports size props too: https://github.com/carbon-design-system/carbon/blob/6a345aabaecef2caaff74d3ddb2e9a503537896b/packages/react/src/components/Toggle/Toggle.js#L61-L64